### ISSUE TITLE
KijijiApiException: One of your images is not on an accepted service. Please upload your images again.

### ISSUE THREAD

**Issue by dudududodododedede** [jackm/kijiji-manager#42]
Suddenly started getting this error when clicking "repost all ads" button (have done this dozens of times without issue)

```
  File "...kijijiapi.py", line 223, in post_ad
    raise KijijiApiException(self._error_reason(doc))
kijiji_manager.kijijiapi.KijijiApiException: One of your images is not on an accepted service. Please upload your images again.
```

**Comment by dudududodododedede** [comment]
Getting the same error when trying to make a new post from scratch as well: "One of your images is not on an accepted service. Please upload your images again."

**Comment by jackm** [comment]
I get the same error and refusal to post the ad. Looks like Kijiji changed their backend for hosting images. I'll have to find out what they changed and update this project accordingly.

**Comment by kobemtl** [comment]
Thank you so much!

**Comment by jackm** [comment]
This has been fixed and released in [v0.1.11](../releases/tag/v0.1.11).
